bug/ci: TestValueExtractorHeaderSource tests fail

**Do you want to request a *feature* or report a *bug*?**

bug

**What is the current behavior?**

CI with Python build flags fails

**What is the expected behavior?**

Tests should pass

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**

```
$ go test -tags "python coprocess" . -run TestValueExtractorHeaderSource
--- FAIL: TestValueExtractorHeaderSource (3.20s)
    --- FAIL: TestValueExtractorHeaderSource/Header_value (1.06s)
    	helpers_test.go:416: [0] Expected status code `200` got `403. {
    		    "error": "Key not authorised"
    		}`. {"Path":"/","Code":200,"Headers":{"Authorization":"valid_token"}}
    	helpers_test.go:416: [1] Expected status code `200` got `403. {
    		    "error": "Key not authorised"
    		}`. {"Path":"/","Code":200,"Headers":{"Authorization":"valid_token"}}
    --- FAIL: TestValueExtractorHeaderSource/Form_value (1.06s)
    	helpers_test.go:416: [0] Expected status code `200` got `403. {
    		    "error": "Key not authorised"
    		}`. {"Method":"POST","Path":"/","Code":200,"Data":"auth=valid_token","Headers":{"Content-Type":"application/x-www-form-urlencoded"}}
    	helpers_test.go:416: [1] Expected status code `200` got `403. {
    		    "error": "Key not authorised"
    		}`. {"Method":"POST","Path":"/","Code":200,"Data":"auth=valid_token","Headers":{"Content-Type":"application/x-www-form-urlencoded"}}
    --- FAIL: TestValueExtractorHeaderSource/Header_regex (1.06s)
    	helpers_test.go:416: [0] Expected status code `200` got `403. {
    		    "error": "Key not authorised"
    		}`. {"Path":"/","Code":200,"Headers":{"Authorization":"prefix-12345"}}
    	helpers_test.go:416: [1] Expected status code `200` got `403. {
    		    "error": "Key not authorised"
    		}`. {"Path":"/","Code":200,"Headers":{"Authorization":"prefix-12345"}}
FAIL
FAIL	github.com/TykTechnologies/tyk	3.274s
```

**Which versions of Tyk affected by this issue? Did this work in previous versions of Tyk?**

Master, v2.7.0, v2.6.2 ...

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
 language: go
 
 notifications:
@@ -14,7 +14,7 @@ addons:
     sources:
     - sourceline: 'ppa:opencpu/jq'
     packages:
-    - python3-dev
+    - python3.5
     - python3-pip
     - libluajit-5.1-dev
     - libjq-dev

